game/glshader.cc

```diff
@@ -162,6 +162,11 @@ Shader& Shader::link() {
 
 	// Link and check status
 	glLinkProgram(program);
+
+	// always detach shaders, linked or not, they need to be detached
+	for (auto id : shader_ids)
```

Comment:
This is more a stylistic concern buy I say either use brackets or don't use a line-break. Not sure what the others think?

Reply:
done